### PR TITLE
Add value thresholds and change wording

### PR DESCRIPTION
**Pull request by peterdavidhamilton**
## Changes in this PR

- Add value thresholds and change wording

## Screenshots of UI changes

skipping screenshots

### CODE REVIEW

**Comment by peterdavidhamilton** [comment]
[Nest steps content tweaks](https://trello.com/c/qokPVeWP/825-nest-steps-content-tweaks)

File: config/static/next_steps_catering.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
 Consider buying from a framework recommended by DfE. This can be quicker and easier than running your own process. Each one has its own list of suppliers and has been assessed for compliance with procurement regulations, ease of use, suitability and value for money.
 
 Go directly to view and choose from a list of <a href="https://find-dfe-approved-framework.service.gov.uk/find/type/on-going/services-categories/facilities/fm-categories/catering" target="_blank">catering frameworks recommended by DfE</a> in the Find a framework tool.
 
 Or <a href="https://www.gov.uk/guidance/buying-procedures-and-procurement-law-for-schools/find-the-right-way-to-buy" target="_blank">read about frameworks and how to use one</a> if you haven't used one before.
 
-## Other procurement options
+## Other compliant procurement options
 
 If you're not able to use a framework, read guidance to find out how to continue with your procurement if you want to:
```
Comment (quaypho) [inline config/static/next_steps_catering.md:15]: This sentence needs replacing with

If you're not able, or choose not to use a framework, make sure you follow a compliant process to continue with your procurement. Read about how to:

**Review by quaypho — CHANGES_REQUESTED** [review]
The sentence above the bullet points needs changing to: If you're not able, or choose not to use a framework, make sure you follow a compliant process to continue with your procurement. Read about how to:

File: config/static/next_steps_mfd.md [hunk 2]
```diff
@@ -1,22 +1,21 @@
...
 Thank you for using this service to create a specification.
 
-You can now share your completed specification with suppliers.
+You can now share your specification with suppliers as part of your invitation to tender (ITT).
 
 Consider buying from a framework recommended by DfE. This can be quicker and easier than running your own process. Each one has its own list of suppliers and has been assessed for compliance with procurement regulations, ease of use, suitability and value for money.
 
 Go directly to view and choose from a list of <a href="https://find-dfe-approved-framework.service.gov.uk/find/type/buying/what/ict/ict-categories/mfd" target ="_blank">multi-functional devices (MFD) and printing frameworks recommended by DfE</a> in the Find a framework tool.
 
 Or <a href="https://www.gov.uk/guidance/buying-procedures-and-procurement-law-for-schools/find-the-right-way-to-buy"target ="_blank">read about frameworks and how to use one</a> if you haven't used one before.
```
Comment (quaypho) [inline config/static/next_steps_mfd.md:11]: The code for this sentence is showing on the front end

**Review by quaypho — CHANGES_REQUESTED** [review]
Not sure why the code seems to be visible - is it missing an asterisks?

**Review by quaypho — APPROVED** [review]
Thanks

**Review by PiperS52 — APPROVED** [review]

**Review by ryantk — APPROVED** [review]

**Pull request merged**